test/minErrSpec.js

```diff
@@ -10,10 +10,29 @@ describe('minErr', function() {
     testError = minErr('test');
 
   var originalObjectMaxDepthInErrorMessage = minErrConfig.objectMaxDepth;
+  var originalUrlMaxLengthInErrorMessage = minErrConfig.urlMaxLength;
   afterEach(function() {
     minErrConfig.objectMaxDepth = originalObjectMaxDepthInErrorMessage;
+    minErrConfig.urlMaxLength = originalUrlMaxLengthInErrorMessage;
   });
 
+  function extractUrlFromErrorMessage(message) {
+    var match = message.match(/http[\s\S]*\?p0=/);
+    var urlStartAt = message.indexOf(match[0]);
+    if (urlStartAt < 0) {
+      throw new Error('Could not find url');
+    }
+    return message.slice(urlStartAt);
+  }
+
+  function createDummyString(size) {
```

Comment:
I don't think we need a helper for this. IIRC, we use `new Array(size).join('a')` elsewhere in the tests.